system/integrations/system-integrations.rst

```diff
@@ -0,0 +1,60 @@
+Integrations
+************
+
+Zammad offers numerous integrations that add rich features to your instance.
+
+.. Note:: We're still working on this part of our documentation, stay tight!
+
+
+Integrations for phone systems
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. toctree::
+   :maxdepth: 1
+   :titlesonly:
+
+   integrations/generic-cti
+   integrations/placetel-cti
+   integrations/sipgate
+   
+Integrations for authentication and customers
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. toctree::
+   :maxdepth: 1
+   :titlesonly:
+
+   integrations/ldap
+   integrations/exchange
+   integrations/clearbit
+   
+Integrations for Monitoring Systems
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. Note:: This section currently misses the following integrations:
+
+  * Check_MK
+  * Icinga
+  * Monit
+  * Nagios
+
+.. toctree::
+   :maxdepth: 1
+   :titlesonly:
+   integrations/zabbix
```

Comment:
Whoops, it also looks like we were missing a newline above this one, which caused the `toctree` directive to fail:

```
system/integrations.rst:41: WARNING: Error in "toctree" directive:
invalid option block.

.. toctree::
   :maxdepth: 1
   :titlesonly:
   integrations/zabbix
 looking for now-outdated files... none found
 pickling environment... done
 checking consistency... /home/rlue/src/zammad-admin-documentation/system/integrations/zabbix.rst: WARNING: document isn't included in any toctree
```